### PR TITLE
Fixing backslash escaping issue

### PR DESCRIPTION
**Pull request by JohanForngren**
Escaping backslash `\` with `&#92;`.

### CODE REVIEW

**Comment by Court72** [comment]
@markingmyname 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by markingmyname — APPROVED** [review]
Thank you for your contribution.

**Comment by markingmyname** [comment]
#sign-off

File: docs/ssms/scripting/search-text-with-regular-expressions.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ The following table describes the regular expressions available in the **Referen
 |Any one character in the set|[]|Matches any one of the characters within the []. To specify a range of characters, list the starting and ending character separated by a dash (-), as in [a-z].|
 |Any one character not in the set|[^...]|Matches any character not in the set of characters following the ^.|
 |Or|&#124;|Matches either the expression before or the one after the OR symbol (&#124;). Mostly used within a group. For example, (sponge&#124;mud) bath matches "sponge bath" and "mud bath."|
-|Escape|\|Matches the character that follows the backslash (\\) as a literal. This allows you to find the characters used in regular expression notation, such as { and ^. For example, \\^ Searches for the ^ character.|
+|Escape|&#92;|Matches the character that follows the backslash (\\) as a literal. This allows you to find the characters used in regular expression notation, such as { and ^. For example, \\^ Searches for the ^ character.|
```
Comment (Jak-MS) [inline docs/ssms/scripting/search-text-with-regular-expressions.md:51]:
```suggestion
|Escape| \\ |Matches the character that follows the backslash (\\) as a literal. This allows you to find the characters used in regular expression notation, such as { and ^. For example, \\^ Searches for the ^ character.|
```
this should render the same without the need for the code

**Pull request merged**